### DISCUSSION TITLE
Minor change to standard Import mechanism readme feature.

### DISCUSSION THREAD

**Discussion by AnthonyMuscio** [TiddlyWiki/TiddlyWiki5#5829] [Ideas]
Folks,

Perhaps we could add a feature to the current import table to detect any inbound tiddlers with the name/.suffix "readme", as comes in plugins or other packages, and open its text field wikified. Thus the readme can be viewed before import so a plugin or package publisher can get the user to read a message before import.

This approach will not only be somewhat intuitive but should work with existing plugin standards, and not cause any backwards compatibility issues.

Regards
Tony

**Comment by pmario** [comment]
How should we deal with a bundle, that contains 2 or more plugins? + the readme of the bundle?

**Reply by AnthonyMuscio** [reply]
Open any readme tiddler in the import tiddler list. 

If a tiddler within the bundle contains more tiddlers eg a plugin ridder, rather than its components no need to make their readme,s visible.
